tests-fuzz/src/ir.rs

```diff
@@ -437,6 +439,42 @@ pub fn generate_columns<R: Rng + 'static>(
         .collect()
 }
 
+/// Replace Value::Default with the corresponding default value in the rows for comparison.
+pub fn replace_default(
```

Comment:
How about changing it to `fill_default_value`? @evenyag

Reply:
Both are ok to me.